lesson01.md

```diff
@@ -1,7 +1,347 @@
+- [Most used programming language](https://www.statista.com/statistics/793628/worldwide-developer-survey-most-used-languages/) among developers worldwide as of 2024
+- [#1 most used language on GitHub](https://octoverse.github.com/2022/top-programming-languages) as of 2022
+
+---
+
+### Tools
+
+We'll be using these tools during our course:
+
+- [Chrome Browser](https://www.google.com/chrome/) (or any modern browser, although Chrome is advised)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [MDN Web Documentation](https://developer.mozilla.org/en-US/)
+
+Please install [Visual Studio Code](https://code.visualstudio.com/)
+
+---
+
+### Additional Material
+
+<!-- .slide: style="font-size:80%" -->
+
+- [javascript.info](https://javascript.info/): tutorials with nice (often visual) explanations
+- [freecodecamp.org](https://www.freecodecamp.org/): full free courses on many topics
+- [udemy.com](https://www.udemy.com): Introduction to JS (course)
+- [codeacademy.com](https://www.codecademy.com): Introduction to JS (another course)
+- [edabit.com](https://edabit.com): Learn JavaScript with interactive challenges and external resources
+- [learnjavascript.online](https://learnjavascript.online): very good online course, but costs 40€
+
+---
+
+### Names of Special Characters
+
+|                                      |                                                               |
+| ------------------------------------ | ------------------------------------------------------------- |
+| `(` Parentheses `)`                  | `<` Angle brackets `>`                                        |
+| `{` Braces, or curly braces `}`      | `&` Ampersand `&`                                             |
+| `[` Brackets, or square brackets `]` | <code>&#124;</code> Vertical bar, or pipe <code>&#124;</code> |
+| `;` Semicolon `;`                    | `:` Colon `:`                                                 |
+| `'` Single quote `'`                 | `#` Hash or number sign `#`                                   |
+| `"` Double quote `"`                 | `` ` `` Back tick `` ` ``                                     |
+| `_` Underscore `_`                   | `*` Asterisk `*`                                              |
+| `/` Slash, or Forward slash `/`      | `~` Tilde `~`                                                 |
+| `\` Backslash `\`                    | `^` Caret or circumflex `^`                                   |
+
+---
+
+### Let's start with JavaScript!
+
+- Open Google Chrome
+- Hit `F12` key
+- _Or opt + cmd + C (Mac) / ctrl + shift + C (Windows)_
+- Click on **Console**
+- Try entering something
+
+What did you try? What did you see?
+
+<!-- .element: class="fragment" -->
+
+---
+
+### Numbers and Numerical Operators
+
+<!-- .slide: style="font-size:80%" -->
+
+Numbers include entire and decimal numbers. In Javascript, we call them integer and float, respectively.
```

Comment:
```suggestion
Numbers include natural and decimal numbers. In Javascript, we call them integer and float, respectively.
```

I couldn't find "entire number" definition, not sure it's a term our students know. Maybe "natural" numbers? In English, they're formally called "integers".